apk-generator/v2/app/utils/notification.py

```diff
@@ -46,7 +81,24 @@ def send_mail_via_smtp_(config, payload):
         mailer.start()
         message = Message(author=payload['from'], to=payload['to'])
         message.subject = payload['subject']
-        message.plain = strip_tags(payload['html'])
-        message.rich = payload['html']
+        message.plain = strip_tags(payload['message'])
+        message.rich = payload['message']
+        message.attach(payload['attachment'], data=None, maintype=None, subtype=None, inline=False)
```

Comment:
I had done this after seeing the Marrow mailer [documentation](https://github.com/marrow/mailer). I think they are required. The maintop can be remove though, I'll do that.